src/test/groovy/org/prebid/server/functional/tests/BidValidationSpec.groovy

```diff
@@ -212,4 +212,34 @@ class BidValidationSpec extends BaseSpec {
         def metrics = defaultPbsService.sendCollectedMetricsRequest()
         assert metrics["adapter.generic.requests.bid_validation"] == initialMetricValue + 1
     }
+
+    def "PBS should return error when between refers to unknown bidder alias"() {
+        given: "Default basic BidRequest"
+        def bidRequest = BidRequest.defaultBidRequest
+        bidRequest.ext.prebid.aliasgvlids = ["between": 1]
```

Comment:
What does `between` signify? Is it a bidder name? Or can it be any string?

Reply:
First of all, do not resolve not yours comments.
That comment wasn't answered here nor resolved in the code.

Reply:
If that is any bidder name, so then use random string generator from PBSUtils to show that test is not bidder specific.